package.json

```diff
@@ -9,6 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
```

Comment:
all @types, all eslint stuff, taiwindcss and autoprefixer are devDependenies uninstall and installed them again as devDependancies